webpack.config.js

```diff
@@ -1 +1,2 @@
 module.exports = require('skatejs-build/webpack.config');
+module.exports.module.loaders[2].query.plugins = ['transform-class-properties'];
```

Comment:
Yep it is. Do you have any suggestions? I was hoping to put this back into skatejs-build prior to this being merged, but didn't want to block if we didn't get around to it.